mozetl/experimentsdaily/fields.py

```diff
@@ -0,0 +1,173 @@
+import pyspark.sql.functions as F
+EXPERIMENT_FIELD_AGGREGATORS = [
```

Comment:
Can you reuse ` mozetl/clientsdaily/fields.py` data? It looks like the only differences are:

```
-  F.first('active_experiment_branch').alias('active_experiment_branch'),
-  F.first('active_experiment_id').alias('active_experiment_id'),
+  F.first('experiment_branch').alias('experiment_branch'),
```